 <launch>

    <group ns="tigra/front" >
        <node pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc"/>

        <!-- Odometry -->
        <node pkg="rtabmap_ros" type="stereo_odometry" name="stereo_odometry" output="screen">

            <param name="frame_id" type="string" value="base_link"/>
            <param name="odom_frame_id" type="string" value="odom_new"/>
            <param name="approx_sync"   type="bool" value="false"/>
            <param name="queue_size"    type="int" value="5"/>

        </node>     
    </group>

    </launch>